modules/serverless-tls-creating-selfsigned-cluster-issuer-eventing.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * /serverless/Eventing/serverless-config-tls-encryption-eventing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-tls-creating-selfsigned-cluster-issuer-eventing_{context}"]
+= Creating a `SelfSigned` `ClusterIssuer` resource for Eventing
+
+`ClusterIssuers` are Kubernetes resources that represent certificate authorities (CAs) capable of generating signed certificates by honoring certificate signing requests. All cert-manager certificates require a referenced issuer in a ready condition to attempt to honor the request. For more details, see link:cert-manager.io/docs/concepts/issuer/[Issuer].
+
+[IMPORTANT]
+====
+For the simplicity, we will use a `SelfSigned` issuer as the root certificate. For more details about `SelfSigned` issuer implications and limitations, see link: https://cert-manager.io/docs/configuration/selfsigned/[SelfSigned issuers]. If you are using a custom public key infrastructure (PKI), you must configure it so its privately signed CA certificates are recognized across the cluster. For more details about cert-manager, see link:https://cert-manager.io/docs/configuration/ca/[certificate authorities (CAs)]. You can use any other issuer that is usable for cluster-local services.
+====
+
+.Prerequisites
+
+* You have cluster administrator permissions on {ocp-product-title}, or you have cluster or dedicated administrator permissions on {rosa-product-title} or {dedicated-product-title}.
+* You have installed the {ServerlessOperatorName}.
+* You have installed the {cert-manager-operator}.
+* You have installed the OpenShift (`oc`) CLI.
+
+.Procedure
+
+. Create a `SelfSigned` `ClusterIssuer` resource as follows:
++
+[source,yaml]
+----
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: knative-eventing-selfsigned-issuer
+spec:
+  selfSigned: {}
+----
+
+. Apply the `ClusterIssuer` resource by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
+
+. Create a root certificate using the `SelfSigned` `ClusterIssuer` resource as follows:
++
+[source,yaml]
+----
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: knative-eventing-selfsigned-ca
+  namespace: cert-manager <1>
+spec:
+  secretName: knative-eventing-ca <2>
+  isCA: true
+  commonName: selfsigned-ca
+  privateKey:
+    algorithm: ECDSA
+    size: 256
+  issuerRef:
+    name: knative-eventing-selfsigned-issuer
+    kind: ClusterIssuer
+    group: cert-manager.io
+----
+<1> It is a {cert-manager-operator} used by default. 
+<2> It is a secret name later used for the `ClusterIssuer` for Eventing.
```

Comment:
```suggestion
<2> It is a secret where the certificate will be stored. The name is later used by the `ClusterIssuer` for Eventing.
```

Reply:
I just want to make clear that here we specify where to store the certificate. The user will not have to wonder where that comes from.

Reply:
Fixed!